data/blog/electron-11-0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Electron 11.0.0
+author:
+- VerteDinde
+- sofianguy
+date: '2020-11-17'
+---
+
+Electron 11.0.0 has been released! It includes upgrades to Chromium `xx`, V8 `xx`, and Node.js `xx`. We've added support for Apple silicon, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 11.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new support for Apple's upcoming silicon products. 
+
+We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `xx`
+    * [New in Chrome 86](https://developers.google.com/web/updates/2020/07/nic84)
+    * [New in Chrome 87](https://chromereleases.googleblog.com/2020/08/stable-channel-update-for-desktop_25.html)
+* Node.js `xx`
+    * [Node 12.16.3 blog post](https://nodejs.org/en/blog/release/v12.16.3/)
+* V8 `8.x`
+    * [V8 8.4 blog post](https://v8.dev/blog/v8-release-84)
+    * [V8 8.5 blog post](https://v8.dev/blog/v8-release-85)
+
+### Highlight Features
+
+* Support for Apple Silicon: As of Electron 11, Electron will be shipping separate versions of Electron for Intel Macs (x64) and Apple's upcoming Silicon hardware (arm64). You can learn more about how to get your Electron app [running on Apple Silicon here.](https://www.electronjs.org/blog/apple-silicon)
+* Minimal changes to Electron APIs: To make upgrading Electron easier for app consumers, the Electron team shipped Electron 11 as a “quiet release”, with minimal changes to Electron APIs. Work for depreciating the remote module and requiring node modules to be context aware continued from previous releases (detailed below).
+
+See the [11.0.0 release notes](https://github.com/electron/electron/releases/tag/v11.0.0) for a full list of new features and changes.
+
+## Breaking Changes
```

Comment:
if we don't have any by the end of this week, we can note that this was intended as a quiet release :) 

> ##Breaking Changes
> Surprise! There wasn't any!
😂

Reply:
oh you mention quiet release in Highlight Features section. ignore me

Reply:
Looks like we didn't have any breaking changes! 🎉 We did have some API additions/improvements, so I highlighted just a few of those as well, to make it clear that some items did change.